.qbt.toml.example

```diff
@@ -13,3 +13,9 @@ interval = 7000 # interval between attempts in milliseconds
 [rules]
 enabled              = true   # enable or disable rules
 max_active_downloads = 2      # set max active downloads
+
+[qbittorrent-compare]
+host     = "100.100.100.100"
+port     = 6776
+login    = "user"
+password = "password"
```

Comment:
Along with changes in config.go, this can be changed to be an array of qbt objects.

```toml
[[compare]]
host     = "100.100.100.100"
port     = 6776
login    = "user"
password = "password"
```

Then you can do this for multiple clients


```toml
[[compare]]
host     = "100.100.100.100"
port     = 6776
login    = "user"
password = "password"

[[compare]]
host     = "100.100.100.101"
port     = 6777
login    = "user"
password = "password"
```